Code/GraphMol/Wrap/rough_test.py

```diff
@@ -3771,7 +3771,7 @@ def testSequenceBasics(self):
     helm = 'PEPTIDE1{C.Y.I.Q.N.C.P.L.G}$$$$'
     seq = 'CYIQNCPLG'
     fasta = '>\nCYIQNCPLG\n'
-    smi = 'CC[C@H](C)[C@H](NC(=O)[C@H](Cc1ccc(O)cc1)NC(=O)[C@@H](N)CS)C(=O)N[C@@H](CCC(N)=O)C(=O)N[C@@H](CC(N)=O)C(=O)N[C@@H](CS)C(=O)N1CCC[C@H]1C(=O)N[C@@H](CC(C)C)C(=O)NCC(=O)O'
+    smi = 'CC[C@H](C)[C@H](NC(=O)[C@H](Cc1ccc(O)cc1)NC(=O)[C@@H](N)CS)C(=O)N[C@@H](CCC(N)=O)C(=O)N[C@@H](CC(N)=O)C(=O)N[C@@H](CS)C(=O)N1CCC[C@@H]1C(=O)N[C@@H](CC(C)C)C(=O)NCC(=O)O'
```

Comment:
I'm pretty sure that the original SMILES here was correct and that you've inverted the stereochemistry of the proline.

Reply:
You are right, it was ok. Apparently, at the time of deciding if the chiral center should be flipped, it did not have an explicit H, but an implicit one, and ended up being flipped.

I have added the implicit valence to the flipping decision, which seems to work fine, and solves this problem.